artilleryTests/rosetta-load-test.yml

```diff
@@ -1,5 +1,7 @@
 config:
-  target: http://localhost:8081
+  target: http://127.0.0.1:8082
+  ensure:
+    maxErrorRate: 1
   http:
     defaults:
```

Comment:
`timeout: 60` should be under the `http:`, not `http: http:`

Reply:
Fixed